waspc/src/Wasp/AppSpec/App/Auth.hs

```diff
@@ -7,6 +7,7 @@ module Wasp.AppSpec.App.Auth
     ExternalAuthConfig (..),
     usernameAndPasswordConfig,
     isUsernameAndPasswordAuthEnabled,
+    isBothExternalAndUsernameAndPasswordAuthEnabled,
```

Comment:
```suggestion
    areBothExternalAndUsernameAndPasswordAuthEnabled,
```

Reply:
For what are you using this variable?

Reply:
That's used for the divider "---------------- Or continue with --------------------" between the social and user/pass login. Otherwise doesn't make sense to add it, since there is nothing to separate.